Source/Media/MediaEngineWrapper.swift

```diff
@@ -27,22 +27,22 @@ class MediaEngineWrapper {
     private var mediaEngineObserver = MediaEngineObserver()
     
     var WMEVersion: String {
-        return MediaEngine.WMEVersion()
+        return MediaEngine.wmeVersion()
     }
 
     init() {
         WmeLogManager.sharedInstance().addLogger(MediaEngineCustomLogger())
         mediaEngineObserver.startObserving()
     }
 
-    func performReachabilityCheck(clusterInfo: [NSObject : AnyObject], completionHandler: ReachabilityCheckHandler) {
-        mediaEngine.performStunReachabilityCheck(clusterInfo) {
+    func performReachabilityCheck(_ clusterInfo: [AnyHashable:Any], completionHandler: @escaping ReachabilityCheckHandler) {
+        mediaEngine?.performStunReachabilityCheck(clusterInfo) {
```

Comment:
I see the MediaEngine.sharedInstance() return Implicitly Unwrapped Optional type, why we still need "?" here?